versions/build.gaeac6.ver

```diff
@@ -1,5 +1,5 @@
 export stack_intel_ver=2023.2.0
 export stack_cray_mpich_ver=8.1.29
-export spack_env=gsi-addon
+export spack_env=gsi-addon-dev-fms-2024.01
```

Comment:
This will impact the entire system. What is the different between `gsi-addon` and `gsi-addon-dev-fms-2024.01`? Just a different fms module available?

Tagging @aerorahul and @DavidHuber-NOAA for their inputs/knowledge.

Reply:
@DavidHuber-NOAA was involved in this, I'll let him respond. 

Various people have run some of the tests with this upgrade on hera, orion, and C6.

Reply:
The `gsi-addon-dev-fms-2024.01` environment is an updated spack-stack 1.6.0 environment.  It is identical to `gsi-addon` except for 3 libraries that I know of: fms, g2, and esmf are all upgraded to newer versions.  This is a chained environment, meaning that it is built off of a parent.  The parent in this case is `fms-2024.01`, which is the environment that the UFS and UFS_Utils are currently using (the `gsi-addon-dev-fms-2024.01` just replaces the `bufr`, `met`, and `metplus` versions as was the case for the `gsi-addon` environment).  On a side note, it may be possible with this upgrade to remove calls to `load_ufswm_modules.sh`, but that would need to be tested and isn't something that should go into this PR.